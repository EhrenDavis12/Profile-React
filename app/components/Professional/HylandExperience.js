import React from 'react';
import { P, SectionBreak } from 'components/SharedStyles/styles';

// const BGIcon = require('images/BGIcon.png');

function HylandExperience() {
  return (
    <div>
      <SectionBreak>
        <h4>Senior Developer</h4>
        <P>APRIL 2013 – CURRENT</P>
        <P>Law Logix; Subdivision of Hyland, Inc.</P>
        <P>
          Industry-leading I-9 and E-Verify Integration Solutions for government
          compliance. Remained with company through buyout in 2016.
        </P>
      </SectionBreak>
      <SectionBreak>
        <ul>
          <li>
            Assisted in architecting, managing, and maintaining an internal flag
            ship product (Guardian)
          </li>
          <li>
            Set up and automated integrations between clients and Guardian.
          </li>
          <li>
            Interfaced with clients to develop strategies and set expectations
          </li>
          <li>
            Independently architected and executed creative solutions to achieve
            custom requests
          </li>
          <li>
            Debugged live issues to created short- and long-term solutions
          </li>
          <li>
            Integrated API’s into primary system to link with 3rd party vendors
          </li>
          <li>
            Participated in code reviews to maintain system integrity, team
            communication, and code continuity
          </li>
          <li>
            Worked with team members to design and architect scoped projects
          </li>
          <li>
            Utilized Jira ticketing system to communicate tasks with other
            developers
          </li>
          <li>
            Continuous documentation of processes and test plans using
            Jira/Confluence
          </li>
          <li>
            Developed and implemented user training documents and schedules
          </li>
          <li>Implemented unit testing to ensure code stability</li>
          <li>
            Created, maintained and ran T-SQL queries to ensure data integrity
          </li>
          <li>
            Created and deployed new releases of internal primary software
          </li>
          <li>Maintained deploy operations to multiple server stacks</li>
          <li>
            Created and Maintain in house tools to streamline company
            automations
          </li>
        </ul>
      </SectionBreak>
    </div>
  );
}

export default HylandExperience;
