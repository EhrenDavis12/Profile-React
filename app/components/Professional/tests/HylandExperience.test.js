import React from 'react';
import { shallow } from 'enzyme';
import 'jest-styled-components';
import HylandExperience from '../HylandExperience';

describe('<HylandExperience />', () => {
  it('should render the page', () => {
    const renderedComponent = shallow(<HylandExperience />);
    expect(renderedComponent).toMatchSnapshot();
  });
});
