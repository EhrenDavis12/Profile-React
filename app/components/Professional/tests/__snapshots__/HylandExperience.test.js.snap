// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HylandExperience /> should render the page 1`] = `
<div>
  <styles__SectionBreak>
    <h4>
      Senior Developer
    </h4>
    <styles__P>
      APRIL 2013 – FEBRUARY 2019
    </styles__P>
    <styles__P>
      Law Logix; Subdivision of Hyland, Inc.
    </styles__P>
    <styles__P>
      Industry-leading I-9 and E-Verify Integration Solutions for government compliance. Remained with company through buyout in 2016.
    </styles__P>
  </styles__SectionBreak>
  <styles__SectionBreak>
    <ul>
      <li>
        Assisted in architecting, managing, and maintaining an internal flag ship product (Guardian)
      </li>
      <li>
        Set up and automated integrations between clients and Guardian.
      </li>
      <li>
        Interfaced with clients to develop strategies and set expectations
      </li>
      <li>
        Independently architected and executed creative solutions to achieve custom requests
      </li>
      <li>
        Debugged live issues to created short- and long-term solutions
      </li>
      <li>
        Integrated API’s into primary system to link with 3rd party vendors
      </li>
      <li>
        Participated in code reviews to maintain system integrity, team communication, and code continuity
      </li>
      <li>
        Worked with team members to design and architect scoped projects
      </li>
      <li>
        Utilized Jira ticketing system to communicate tasks with other developers
      </li>
      <li>
        Continuous documentation of processes and test plans using Jira/Confluence
      </li>
      <li>
        Developed and implemented user training documents and schedules
      </li>
      <li>
        Implemented unit testing to ensure code stability
      </li>
      <li>
        Created, maintained and ran T-SQL queries to ensure data integrity
      </li>
      <li>
        Created and deployed new releases of internal primary software
      </li>
      <li>
        Maintained deploy operations to multiple server stacks
      </li>
      <li>
        Created and Maintain in house tools to streamline company automations
      </li>
    </ul>
  </styles__SectionBreak>
</div>
`;
